subprojects/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc

```diff
@@ -0,0 +1,336 @@
+
+Enter target version of Java (min. 7) (default: 19):
+Generate build using new APIs and behavior (some features may change in the next minor release)? (default: no) [yes, no]
+----
+
+NOTE: In this tutorial, Kotlin DSL is used to build a simple Java project (as it is the default DSL starting in Gradle 8.2). All examples are macOS based.
+
+== Step 2. Understanding the Directory layout
+
+The *project root directory* contains all source files from your project.
+
+When you are done with Gradle `init`, the directory should look as follows:
+
+----
+.
+├── gradle                              // <1>
+│   └── wrapper
+├── gradlew                             // <2>
+├── gradlew.bat                         // <2>
+├── settings.gradle.kts                 // <3>
+└── app
+    ├── build.gradle.kts                // <4>
+    └── src
+        ├── main
+        │   └── java
+        │       └── demo
+        │           └── App.java        // <5>
+        └── test
+            └── java
+                └── demo
+                    └── AppTest.java    // <6>
+----
+<1> Generated folder for wrapper files
+<2> Gradle wrapper start scripts
+<3> Settings file to define build name and subprojects
+<4> Build script for `app` dependencies
+<5> Default Java source folder for `app`
+<6> Default Java test source folder for `app`
+
+The `authoring-tutorial` folder is the *root project directory*.
+Inside the root project directory are one or more subprojects, build scripts, and the Gradle wrapper.
+
+While the Gradle Wrapper is local to the root project, the Gradle executable is found in the `GRADLE_USER_HOME`.
+
+The `GRADLE_USER_HOME`, which defaults to `USER_HOME/.gradle`, is also where Gradle stores its global configuration properties, initialization scripts, caches, log files and more.
+
+== Step 3. Review the Gradle Files
+
+The `settings.gradle.kts` file has two interesting lines:
+
+.settings.gradle.kts
+[source, kotlin]
+----
+rootProject.name = "authoring-tutorial"
+include("app")
+----
+
+- `rootProject.name` assigns a name to the build, which overrides the default behavior of naming the build after the directory it’s in.
+- `include("app")` defines that the build consists of one subproject called `app` that contains its own source code and build logic.
+
+More subprojects can be added by additional `include()` statements.
+
+Our build contains one subproject called `app` that represents the Java application we are building.
+It is configured in the `app/build.gradle(.kts)` file:
+
+.build.gradle.kts
+[source, kotlin]
+----
+plugins {
+    id("application")                                               // <1>
+}
+
+repositories {
+    mavenCentral()                                                  // <2>
+}
+
+dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter:5.9.3")     // <3>
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+    implementation("com.google.guava:guava:32.1.1-jre")             // <4>
```

Comment:
🤔 Where does this (or Running Gradle Builds) discuss what `implemetnation` and `testImplementation` mean?  This doesn't attempt to teach this here, so I assume it's meant to be explained elsewhere.

Reply:
Explained in part 5.